### PR TITLE
Redesign CSA page

### PR DESCRIPTION
**Pull request by kelbyhawn**
Redesign the CSA page https://code.org/educate/csa

- Page is fully translatable and responsive

Jira ticket: [ACQ-624](https://codedotorg.atlassian.net/browse/ACQ-624)

----

### Before
Before

### After
After

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/educate/csa.haml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+  .wrapper
+    = view :curriculum_glance_section, heading: hoc_s(:heading_curriculum_at_a_glance), grades: hoc_s(:module_grade_9_12), duration: hoc_s(:module_duration_school_year), level: hoc_s(:module_level_intermediate), tools: hoc_s(:csa_page_curriculum_glance_tools), devices: hoc_s(:csa_page_curriculum_glance_devices), professional_learning: hoc_s(:csa_page_curriculum_glance_professional_learning), topics: hoc_s(:csa_page_curriculum_glance_topics), accessibility_features: hoc_s(:csa_page_curriculum_glance_accessibility_features), languages: hoc_s(:csa_page_curriculum_glance_languages)
+
+%section.quote.bg-neutral-dark
+  .wrapper
+    %div.quotes-wrapper
+      %p.quote-text
+        =hoc_s(:csa_page_quote)
+      %p.no-margin-bottom
+        -# %span •
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/educate/csa.haml:47]: Commenting this out for now since we don't have a teacher name

**Review by dmcavoy — APPROVED** [review]
Product approval - note you can't get into Javalab level without signing in and maybe not even when signed in if you aren't verified but I think its ok for now. I will follow up with Mike O and Jamila

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml [hunk 2]
```diff
@@ -177,6 +183,8 @@
   call_to_action_see_module_details: "See module details"
   call_to_action_start_modules: "Start modules"
   call_to_action_see_course_details: "See course details"
+  call_to_action_learn_about_approach: "Learn more about our approach"
+  call_to_action_try_java_lab: " Try Java Lab"
```
Comment (TurnerRiley) [inline pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Is that space at the beginning of " Try Java Lab" intended?

**Review by TurnerRiley — APPROVED** [review]
On a tear again Kelby! This looks great! 1 tiny string question

File: pegasus/sites.v3/hourofcode.com/i18n/en.yml (hunk 2 above)
Comment (kelbyhawn) [reply pegasus/sites.v3/hourofcode.com/i18n/en.yml]: Nope, just fixed! Do you have a method/tool that helps find whitespace errors?

**Pull request merged**